tests/passw_hardening/test_passw_hardening.py

```diff
@@ -88,6 +87,25 @@ def get_passw_expire_time_existing_user(duthost, normal_account):
     return last_passw_change
 
 
+def check_expiration_value(duthost, expiration_value):
+    """
+    Determine whether last password change date will be expired compared
+    to expiration value to be set
+    Args:
+        duthost: duthost object
+        expiration_value: expiration value to be checked
+    Return:
+        expiration: Expiration value to be set
+    """
+    msg = "Expiration diff is {}; Expected expiration to set {}; expiration used {}"
+    last_passw_change = duthost.shell('chage -l `echo "$USER"` | head -1')['stdout']
```

Comment:
Is there a case that the date/time is not available?

Reply:
It doesn't look like there is such a case, Last password change date is present after adding new user already:
```
$ sudo useradd test_a
$ sudo chage -l test_a | head -1
Last password change                            : Oct 27, 2022
```